extensions/agroal/deployment/src/test/resources/application-multiple-datasources.properties

```diff
@@ -6,7 +6,7 @@ quarkus.datasource.jdbc.max-size=13
 
 quarkus.datasource.users.db-kind=h2
 quarkus.datasource.users.username=username1
-quarkus.datasource.users.jdbc.driver=org.h2.Driver
+quarkus.datasource.users.driver=org.h2.Driver
```

Comment:
Why this change, you are creating an invalid configuration for a test that was perfectly valid?

Reply:
I wanted to show that even with this invalid configuration, things now work. If you try to run the this configuration without the PR, the test will fail with an obscure Arc error.
I can of course duplicate this test if you like to leave the original test intact.

Reply:
Yeah, I would prefer to keep the test as is. If you want to test that, please add another test.

Reply:
PR updated